<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Anshuls-MacBook.local" ignored="0" name="Listener Test Case" tests="5" failures="1" timestamp="2022-06-14T19:49:38 IST" time="0.093" errors="0">
  <testcase name="testFour" time="0.012" classname="com.testng.listeners.SampleTestClass">
    <skipped/>
  </testcase> <!-- testFour -->
  <testcase name="testGetData" time="0.002" classname="com.testng.listeners.SampleTestClass"/>
  <testcase name="testGetData" time="0.002" classname="com.testng.listeners.SampleTestClass"/>
  <testcase name="testOne" time="0.002" classname="com.testng.listeners.SampleTestClass"/>
  <testcase name="testThree" time="0.009" classname="com.testng.listeners.SampleTestClass">
    <failure type="java.lang.AssertionError" message="expected [7] but found [5]">
      <![CDATA[java.lang.AssertionError: expected [7] but found [5]
at com.testng.listeners.SampleTestClass.testThree(SampleTestClass.java:26)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testThree -->
</testsuite> <!-- Listener Test Case -->
